README.rst

```diff
@@ -1,7 +1,9 @@
-##############
+==============
 analysis_tools
-##############
+==============
 
-``analysis_tools`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+The Analysis Tools package is designed to assist in the creation of quality assurance (QA) plots and metrics from the outputs of a data reduction pipeline.
+The intention is that users have the flexibility to construct complex data analysis tasks from a set of simple building blocks.
+In this sense, a series of consistent, repeatable and high-quality plots and metrics can be generated for any given dataset.
```

Comment:
"In this sense"? Maybe something like this:

```suggestion
Analysis Tools allows users to generate consistent, repeatable, and high-quality plots and metrics on the output of a PipelineTask, no matter which dataset it is run on.
```

Reply:
Barring the pipeline task issue above, this is 👍